airbyte-cdk/python/unit_tests/sources/file_based/in_memory_files_source.py

```diff
@@ -48,7 +48,7 @@ def get_matching_files(
         from_date: Optional[datetime] = None,
```

Comment:
Is there a reason we don't use `from_date` here? Shouldn't we filter files based on `last_modified`?